pandas/core/tools/datetimes.py

```diff
@@ -854,16 +914,93 @@ def to_datetime(
     DatetimeIndex(['1960-01-02', '1960-01-03', '1960-01-04'],
                   dtype='datetime64[ns]', freq=None)
 
-    In case input is list-like and the elements of input are of mixed
-    timezones, return will have object type Index if utc=False.
+    **b. Non-convertible date/times**
 
-    >>> pd.to_datetime(['2018-10-26 12:00 -0530', '2018-10-26 12:00 -0500'])
-    Index([2018-10-26 12:00:00-05:30, 2018-10-26 12:00:00-05:00], dtype='object')
+    If a date does not meet the `timestamp limitations
+    <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html
+    #timeseries-timestamp-limits>`_, passing errors='ignore'
+    will return the original input instead of raising any exception.
+
+    Passing ``errors='coerce'`` will force an out-of-bounds date to ``NaT``,
+    in addition to forcing non-dates (or non-parseable dates) to ``NaT``.
+
+    >>> pd.to_datetime('13000101', format='%Y%m%d', errors='ignore')
+    datetime.datetime(1300, 1, 1, 0, 0)
+    >>> pd.to_datetime('13000101', format='%Y%m%d', errors='coerce')
+    NaT
+
+    .. _to_datetime_tz_examples:
+
+    **c. Timezones and time offsets**
+
+    The default behaviour (``utc=False``) might be confusing concerning timezones:
```

Comment:
Don't think this sentence adds much. I think the bullets above each example are sufficient

Reply:
This is to oppose the first group of bullets (utc=False) with the end of the subsection (utc=True).
Would replacing this with 

    `The default behaviour (``utc=False``) is as follows:`

be better ?

Reply:
Yes that would be better thanks